apps/shared/avifjpeg.c

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: BSD-2-Clause
 
 #include "avifjpeg.h"
-#include "avif/internal.h"
+#include "avifexif.h"
```

Comment:
Nit: List headers in alphabetical order.

Make the same change to apps/shared/avifpng.c.

Reply:
To make that clearer we could add blank line:

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes